OpenOversight/app/templates/officer.html

```diff
@@ -1,14 +1,92 @@
 {% extends "base.html" %}
+{% block title %}{{ officer.full_name() }} - OpenOversight{% endblock %}
+{% block meta %}
+  {% set job_title = officer.assignments[0].job.job_title if officer.assignments[0] and officer.assignments[0].job.job_title != 'Not Sure' else 'Employee' %}
```

Comment:
*nitpick* Can we simplify this line now too using the `officer.job_title()`? Still a little awkward since we have to replace both `None` and `'Not Sure'` with `'Employee'`, so maybe it can wait for when we get rid of the 'Not Sure' case everywhere.